src/activemasternode.cpp

```diff
@@ -154,29 +165,57 @@ void CActiveMasternode::ManageStateInitial()
         return;
     }
 
+    // Default to REMOTE
+    eType = MASTERNODE_REMOTE;
```

Comment:
That's too early, remote masternode needs a wallet to be enabled and unlocked too otherwise it won't be able to sign PS transactions iirc. By assigning it here you only verify it once, (which could) fail and skip right to `ManageStateRemote` next time which is no good.